libraries/lib-track/Track.cpp

```diff
@@ -44,8 +44,9 @@ Track::Track()
    mOffset = 0.0;
 }
 
-Track::Track(const Track &orig, ProtectedCreationArg&&)
-: vrulerSize( orig.vrulerSize )
+Track::Track(const Track& orig, ProtectedCreationArg&&)
+    : vrulerSize { orig.vrulerSize }
+    , mProjectTempo { orig.mProjectTempo }
```

Comment:
Expect a small conflict if you rebase this onto my Clip-views branch.  I eliminate `Track::vrulerSize`.